source/support.html.erb

```diff
@@ -90,18 +95,17 @@ title: Support
       <h2 class="heading-large">Contacting us about a critical issue</h2>
 
       <p>
-        To contact us about a critical issue during working hours, please use
-        our support email address,
+        To contact us during working hours, use our support email address,
         <a href="mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk">
           gov-uk-paas-support@digital.cabinet-office.gov.uk
         </a>.
       </p>
 
       <p>
-        To contact us outside of working hours, please use the emergency email
-        address you have been given. This will raise a high priority alert and
-        could wake us up in the middle of the night, so it should only be used
-        for critical incidents.
+        To contact us about a critical issue outside of working hours, please
+        use the emergency email address you have been given. This will raise a
+        high priority alert and could wake us up in the middle of the night, so
```

Comment:
The thing we're trying to highlight here is the impact using this email address will have on the team, and I'm not sure your suggestion quite does this. We could say something like: 'This will trigger a high priority ticket which could alert the team during the night, so it should only be used for critical incidents.'